integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.testcases.standalone.apiDeploy;
+
+import com.github.dockerjava.zerodep.shaded.org.apache.hc.core5.http.HttpStatus;
+import com.google.common.net.HttpHeaders;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.choreo.connect.mockbackend.ResponseConstants;
+import org.wso2.choreo.connect.tests.context.CCTestException;
+import org.wso2.choreo.connect.tests.util.HttpResponse;
+import org.wso2.choreo.connect.tests.util.HttpsClientRequest;
+import org.wso2.choreo.connect.tests.util.TestConstant;
+import org.wso2.choreo.connect.tests.util.TokenUtil;
+import org.wso2.choreo.connect.tests.util.Utils;
+
+import java.net.MalformedURLException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TrailingSlashTestCase {
+    private String jwtTokenProd;
+    private Map<String, String> headers = new HashMap<>();
+    private static final String API_CONTEXT = "/trailing-slash";
+
+    @BeforeClass(description = "Get Prod token")
+    void start() throws Exception {
+        jwtTokenProd = TokenUtil.getJwtForPetstore(TestConstant.KEY_TYPE_PRODUCTION, null, false);
+        headers.put(HttpHeaders.AUTHORIZATION, "Bearer " + jwtTokenProd);
+    }
+
+    @Test(description = "test path with trailing slash but without path parameters")
+    public void testTrailingSlashWithoutPathParam() throws CCTestException, MalformedURLException {
+        String endpoint = Utils.getServiceURLHttps(API_CONTEXT + "/pet/findByStatus");
```

Comment:
Need the same test case here also.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2877/commits/b5b41e57906ff728fcda6d305ef7195b7e992146 and https://github.com/wso2/product-microgateway/pull/2877/commits/10431d907d6974a745483c45c5b7a89aad3eec10